x-pack/plugins/cases/server/services/cases/index.ts

```diff
@@ -503,6 +493,8 @@ export class CasesService {
             username,
             full_name: user.full_name ?? null,
             email: user.email ?? null,
+            // TODO: verify that adding a new field is ok, shouldn't be a breaking change
+            profile_uid: user.profile_uid,
```

Comment:
@cnasikas I believe adding new fields to an existing API is ok right?

Reply:
Yes, extending the API contract is fine.